#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-defaults.dpatch by Christian Welzel <gawain@camlann.de>
##
## DP: provide some default values for Debian systems

@DPATCH@

--- typo3-dummy-4.0.2.orig/typo3conf/localconf.php	2006-09-11 09:18:23.000000000 +0200
+++ typo3-dummy-4.0.2/typo3conf/localconf.php	2006-09-23 18:30:33.072210250 +0200
@@ -10,4 +10,74 @@
 
 ## INSTALL SCRIPT EDIT POINT TOKEN - all lines after this points may be changed by the install script!
 
-?>
\ Kein Zeilenumbruch am Dateiende.
+// If safe_mode is activated with TYPO3, disable use of
+// external programs
+// typo3-dummy provides links to the most important external programs in
+// /var/lib/typo3-dummy/execdir, so exec() function is enabled by default.
+$TYPO3_CONF_VARS["BE"]["disable_exec_function"] = '0';
+
+// TYPO3 prefers the use of GIF-files and most likely your visitors on
+// your website does too as not all browsers support PNG yet.
+$TYPO3_CONF_VARS["GFX"]["gdlib_png"] = '1';
+
+// set this true to get some better results in GIFBUILDER
+$TYPO3_CONF_VARS['GFX']['png_truecolor'] = '1';
+
+// enabling the use of gdblib2 for image processing
+$TYPO3_CONF_VARS["GFX"]["gdlib_2"] = '1';
+
+// last versions of imagemagick renamed combine to composite. It will
+// be set by basic configuration automatically.
+$TYPO3_CONF_VARS["GFX"]["im_combine_filename"] = 'combine';
+
+// This value should be set to 1 if imagemagick version is greater
+// than 5.2
+$TYPO3_CONF_VARS["GFX"]["im_negate_mask"] = '1';
+
+$TYPO3_CONF_VARS["GFX"]["im_imvMaskState"] = '1';
+
+$TYPO3_CONF_VARS["GFX"]["im_mask_temp_ext_gif"] = '1';
+
+// The value should be 0 if the version of imagemagick is greater than
+// 5, otherwise the creation of effects is getting too slow
+$TYPO3_CONF_VARS["GFX"]["im_no_effects"] = '1';
+
+$TYPO3_CONF_VARS["GFX"]["im_v5effects"] = '1';
+
+// Path to the imagemagick manipulation tools like convert,
+// composite and identify
+$TYPO3_CONF_VARS["GFX"]["im_path"] = '/var/lib/typo3-dummy/execdir/';
+$TYPO3_CONF_VARS['GFX']["im_path_lzw"] = '/var/lib/typo3-dummy/execdir/';
+
+// Set Value to 1 if version of ImageMagick is greater than 4.9
+$TYPO3_CONF_VARS["GFX"]["im_version_5"] = 'gm';
+
+// This variable can be empty if ImageMagick is compiled with LZW.
+// Otherwise you have to set the path to LZW
+//$TYPO3_CONF_VARS["GFX"]["im_path_lzw"] = '';
+
+// Image file formats that should be accepted by Typo3
+$TYPO3_CONF_VARS["GFX"]["imagefile_ext"] = 'gif,jpg,jpeg,tif,bmp,pcx,tga,png,pdf';
+
+$TYPO3_CONF_VARS["GFX"]["im_noFramePrepended"] = '1';
+
+// Enables the preview of images to make the choice more easy
+$TYPO3_CONF_VARS["GFX"]["thumbnails"] = '1';
+
+// Preview of images in png or gif format.
+// Should be the same as "gdlib_png"
+$TYPO3_CONF_VARS["GFX"]["thumbnails_png"] = '1';
+
+// Check freetype quicktest in the basic configuration if text is
+// exceeding the image borders. If yes, you are using Freetype 2 and
+// need to set TTFdpi to 96 dpi
+$TYPO3_CONF_VARS["GFX"]["TTFdpi"] = '96';
+
+// set memory limit to 48 mb.
+// You may want to set this limit higher to get the extension manager working.
+$TYPO3_CONF_VARS['SYS']['setMemoryLimit'] = '48';
+
+// Defines which of these PHP-features to use for various Charset conversing 
+// functions in t3lib_cs.
+$TYPO3_CONF_VARS['SYS']['t3lib_cs_convMethod'] = 'iconv';
+?>
